Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -1549,15 +1548,15 @@ static ErrorOr<void> create_code_point_tables(UnicodeData& unicode_data)
             auto property_names = property_list.keys();
             quick_sort(property_names);
 
-            for (auto const& property_name : property_names) {
-                auto const& code_point_rages = property_list.get(property_name).value();
-                data.property_values.unchecked_append(code_point_rages);
+            for (auto& property_name : property_names) {
+                auto& code_point_rages = property_list.get(property_name).value();
```

Comment:
I think this is meant to be "ranges"? Though too much unicode can do that to a person. 😡